ci/style.sh

```diff
@@ -9,7 +9,7 @@ if [ -n "${CI:-}" ]; then
     check="--check"
 fi
 
-rustc ci/style.rs && ./style src
+cargo test --manifest-path libc-test/Cargo.toml --test style
```

Comment:
Good point, though I think this conflicts with the root directory arg that is passed in. Is there anyway we can have both or maybe we should just always run from `src`?

https://github.com/rust-lang/libc/blob/55476a052cd37b1855b05767333ecc569cf87ee9/libc-test/test/style.rs#L56

Reply:
I don't think there is any need to retain the ability to pass a path, especially if it can figure out the path relative to `CARGO_MANIFEST_DIR`. Wouldn't the test harness be eating that arg anyway

Reply:
Sounds good! It seems like getting the args in the test takes priority over the test harness so it could be useful for just checking a subdir but I don't think this is very common anyways